### ISSUE TITLE
support dynamic imports ?

### ISSUE THREAD

**Issue by jackie19** [ducksoupdev/vue-webpack-typescript#21]
https://webpack.js.org/guides/code-splitting/

**Comment by filipac** [comment]
👍 +1 on that. I tried to make it work, it creates a separate module file but then i get this error:
```
[Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <Anonymous>
       <Root>
```

**Comment by ducksoupdev** [comment]
I use dynamic imports for defining routing. Here is an example.

```ts
import Vue from "vue";
import VueRouter, { Location, Route, RouteConfig } from "vue-router";

const homeComponent = () => import("./components/home").then(({ HomeComponent }) => HomeComponent);

Vue.use(VueRouter);

export const createRoutes: () => RouteConfig[] = () => [
    {
        path: "/",
        component: homeComponent,
    }
];
```

The `homeComponent` is split automatically by Webpack.

**Comment by ducksoupdev** [comment]
Implemented in v1.1.0
